test/test.baseexternalclient.ts

```diff
@@ -82,6 +82,25 @@ describe('BaseExternalAccountClient', () => {
     client_id: 'CLIENT_ID',
     client_secret: 'SECRET',
   };
+  const externalAccountOptionsWorkforceUserProject = Object.assign(
+    {
+      workforce_pool_user_project: 'work_force_pool_user_project',
+    },
+    externalAccountOptions
+  );
+  externalAccountOptionsWorkforceUserProject.audience =
+    '//iam.googleapis.com/projects/projectId/locations/global/workforcePools/pool/providers/provider';
+  externalAccountOptionsWorkforceUserProject.subject_token_type =
+    'urn:ietf:params:oauth:token-type:id_token';
+
+  const externalAccountOptionsWithClientAuthAndWorkforceUserProject =
+    Object.assign(
+      {
+        client_id: 'CLIENT_ID',
+        client_secret: 'SECRET',
+      },
+      externalAccountOptionsWorkforceUserProject
+    );
```

Comment:
nit: these are used once, why not inline?

Reply:
https://github.com/googleapis/google-auth-library-nodejs/blob/3ab1fc46610a090b8a207790b3a1d9f116b3c8a1/test/test.baseexternalclient.ts#L114,
just trying to follow code style, this is once used once also